README.md

```diff
@@ -126,15 +125,33 @@ The result object has:
 You can manually reset a fill a bucket using PUT:
 
 ```js
-limitd.put({ type: 'ip', key: '54.21.23.12' }, err => {});
+limitd.put(type, key, [count], (err, result) => {
+  console.log(result);
+});
 ```
 
-`limitd.put` takes as argument an object with:
+`limitd.put` takes the following arguments:
 
 -  `type`: the bucket type.
 -  `key`: the identifier of the bucket.
 -  `count`: the amount of tokens you want to put in the bucket. This is optional and the default is the size of the bucket.
 
+## STATUS
+
+_Warning_: This is method's performance is not guaranteed.
```

Comment:
Does it degrades the whole system perf? (i.e. other calls by increasing load)

Reply:
Not necessarily. If you are using cluster, it will end up in slaves if available.